docs/src/routes/documentation/NodeGroup/Example5.js

```diff
@@ -0,0 +1,171 @@
+// @flow weak
+/* eslint react/no-multi-comp: "off", max-len: "off" */
+
+import { interpolate } from 'd3-interpolate';
+import { scaleOrdinal } from 'd3-scale';
+import { arc, pie } from 'd3-shape';
+import Surface from 'docs/src/components/Surface';
+import React, { PureComponent } from 'react';
+import NodeGroup from 'resonance/NodeGroup';
+
+const colors = scaleOrdinal()
+  .range([
+    '#6C6B74',
+    '#2E303E',
+    '#9199BE',
+    '#54678F',
+    '#212624',
+    '#3A4751',
+    '#1E272E',
+    '#0A151D',
+    '#030C12',
+    '#253517',
+    '#5D704E',
+    '#324027',
+    '#19280C',
+    '#0D1903',
+  ]);
+
+// **************************************************
+//  SVG Layout
+// **************************************************
+const view = [1000, 250];      // [width, height]
+const trbl = [10, 10, 10, 10]; // [top, right, bottom, left] margins
+
+const dims = [ // Adjusted dimensions [width, height]
+  view[0] - trbl[1] - trbl[3],
+  view[1] - trbl[0] - trbl[2],
+];
+
+// **************************************************
+//  Mock Data
+//  Data is being generated by Math.random in the
+//  constructor and randomized in update
+// **************************************************
+const mockData = [
+  {
+    name: 'Linktype',
+  }, {
+    name: 'Quaxo',
+  }, {
+    name: 'Skynoodle',
+  }, {
+    name: 'Realmix',
+  }, {
+    name: 'Jetpulse',
+  }, {
+    name: 'Chatterbridge',
+  }, {
+    name: 'Riffpedia',
+  }, {
+    name: 'Layo',
+  }, {
+    name: 'Oyoba',
+  }, {
+    name: 'Ntags',
+  },
+];
+
+const getArc = pie()
+  .value((d) => d.value)
+  .sort(null);
+
+const arcPath = arc()
+  .innerRadius((dims[1] / 3.5))
+  .outerRadius((dims[1] / 2));
+
+class Example extends PureComponent {
+  constructor(props) {
+    super(props);
+
+    const data = mockData.map((n) => {
+      return { ...n, value: Math.random() };
+    });
+
+    const newArcs = getArc(data);
+    this.state = {
+      data,
```

Comment:
I did this in my example, too.  We're not actually using the data, so there's no need to keep it in state.